docs/configuration/config-file-reference.md

```diff
@@ -4245,6 +4255,21 @@ ring:
 # CLI flag: -experimental.ruler.enable-api
 [enable_api: <boolean> | default = false]
 
+# EXPERIMENTAL: Enable rulers to store a copy of rules owned by other rulers
+# with default state (state before any evaluation) and send this copy in list
+# API requests as backup in case the ruler who owns the rule fails to send its
+# rules. This allows the rules API to handle ruler outage by returning rules
+# with default state. Ring replication-factor needs to be set to 3 or more for
```

Comment:
Nit: recommend to have replication-factor to more than 3?

Reply:
Not sure I understand, why would we recommend to have a replication factor more than 3?

Reply:
Why it needs to be 3? I understand for Ruler API HA, 2 should be good enough. You don't have to require quorum to have 2 > 1 since you are merging the response anyway.